Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM golang:1.18-alpine AS builder
+RUN apk update && apk add --no-cache make git gcc libc-dev linux-headers python3-dev
+WORKDIR /nucleus
+COPY . /nucleus
+RUN rm -rf .git && git init
+RUN go mod tidy && go mod download
+RUN make clean & make build
+RUN python3 ./init.py -b /nucleus/build/nucleusd -c /root/.nucleus --overwrite 
+COPY addrbook.json /root/.nucleus/config/addrbook.json
```

Comment:
Not every use case requires `init.py` to be run. We should provide a plain nucleus daemon and leave the rest to the users. How about simply copying `init.py` (without running) into the container?

Also `addrbook.json` file can be moved into https://github.com/KomodoPlatform/nucleus-assets repository.

Reply:
copying init.py into container will mean needing to add some extra py related deps, and it wont work unless we also copy the app.toml template.